tidb-cloud/delete-tidb-cluster.md

```diff
@@ -17,19 +17,23 @@ You can delete a cluster at any time by performing the following steps:
     > Alternatively, you can also click the name of the target cluster to go to its overview page, and then click **...** in the upper-right corner.
 
 3. Click **Delete** in the drop-down menu.
-4. In the cluster deleting window, enter your `<organization name>/<project name>/<cluster name>`.
+4. In the cluster deleting window, the prompt message will inform you of the number of backups and the charging policy for backups if you have at least one manual or automatic backups. If you click **Continue** or if no backups are present, you will be prompted to enter `<organization name>/<project name>/<cluster name>` to confirm the deletion.
 
     If you want to restore the cluster sometime in the future, make sure that you have a backup of the cluster. Otherwise, you cannot restore it anymore. For more information about how to back up TiDB Dedicated clusters, see [Back Up and Restore TiDB Dedicated Data](/tidb-cloud/backup-and-restore.md).
 
     > **Note:**
     >
     > [TiDB Serverless clusters](/tidb-cloud/select-cluster-tier.md#tidb-serverless) only support [in-place restoring from backups](/tidb-cloud/backup-and-restore-serverless.md#restore) and do not support restoring data after the deletion. If you want to delete a TiDB Serverless cluster and restore its data in the future, you can use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export your data as a backup.
 
-5. Click **I understand the consequences. Delete this cluster**.
+5. Click **I understand, delete it**.
 
  Once a backed up TiDB Dedicated cluster is deleted, the existing backup files of the cluster are moved to the recycle bin.
 
-- For backup files from an automatic backup, the recycle bin can retain them for 7 days.
-- For backup files from a manual backup, there is no expiration date.
+- Automatic backups will expire and be automatically deleted once the retention period has passed. The default retention period is 7 days if you don't modify it. 
+- Manual backups will be kept in the Recycle Bin until manually deleted.
```

Comment:
```suggestion
- Automatic backups will stop and be automatically deleted once the retention period expires. The default retention period is 7 days if you don't modify it.
- Manual backups will be kept in the Recycle Bin until manually deleted.
```

Reply:
Thanks for comment, but I think stop is not accurate for automatic backup. It's not stop backup but backup files will be expired when the retention time arrived.

Reply:
Got the meaning. Then let's keep "expire".